cosineSimilarity/Cosine.py

```diff
@@ -0,0 +1,23 @@
+from sklearn.feature_extraction.text import TfidfVectorizer
+from nltk.corpus import stopwords
+import numpy as np
+import numpy.linalg as LA
+
+cosine_function = lambda a, b : round(np.inner(a, b)/(LA.norm(a)*LA.norm(b)), 3)
+
+documents = ["The sky is blue.", "The sun is bright."]
+query = ["The sun in the sky is bright."]
+stopWords = stopwords.words('english')
```

Comment:
You should put these under a `if __name__ == '__main__'` so that it's a test and if it is imported, this won't cause issues

Reply:
i don't really know what you mean. i can hop on a call tonight/tomorrow to discuss this with you if you'd like.

Reply:
I am free the rest of the night for this if desired.